flight-reservation/components/CheckBoxComponent/index.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+import { View  } from "react-native";
+import React, { useState } from "react";
+import BouncyCheckbox from "react-native-bouncy-checkbox";
+import { styles } from "./styles";
+
+
+const CheckBoxComponent = ({text, status}) => {
+  // for later use
+  // const [checked, setChecked] = useState(false);
+  // const [color, setColor] = useState('gray')
+
```

Comment:
If you're not using code, it's a good idea to remove it from GitHub.

Remember that it is code that is intended for production.